.changeset/sour-pears-think.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/admin.organizations.v1": patch
+"@wso2is/admin.tenants.v1": minor
+"@wso2is/i18n": minor
```

Comment:
We don't need to release a minor version of i18n for this change, do we?

Reply:
I've added the `advanced search` related strings to the package. So it's a feature addition and not a bug fix.
So it should be a `minor` right?

Reply:
Ack